net-analyzer/mk-livestatus/mk-livestatus-1.2.2_p3-r1.ebuild

```diff
@@ -0,0 +1,130 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+
+GENTOO_DEPEND_ON_PERL=no
+PERL_EXPORT_PHASE_FUNCTIONS=no
+PYTHON_COMPAT=( python2_7 )
```

Comment:
Could you test compatibility with Python 3? With more then one python implementation the use of the python-r1 eclass would make more sense.